bigbluebutton-html5/imports/ui/components/notifications/container.jsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
 import { Notifications as NotificationsCollection } from '/imports/api/meetings';
 import { notify } from '/imports/ui/services/notification';
 import { withTracker } from 'meteor/react-meteor-data';
+import WaitingUsersAlertService from '/imports/ui/components/waiting-users/alert/service';
+import UserService from '/imports/ui/components/user-list/service';
 
-export default withTracker(() => {
+export default injectIntl(withTracker(({ intl }) => {
   NotificationsCollection.find({}).observe({
     added: (obj) => {
+      if (obj.messageId === 'app.userList.guest.pendingGuestAlert') {
```

Comment:
Should the NotificationsCollection.find filter by meetingId? An edge case would be receiving breakout1 annotations in the main room or vice versa

Reply:
this is handled by the publisher (https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/api/meetings/server/publishers.js#L143-L154)